src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int FILE_NAME = 0;
+    private static final int RESULT_NAME = 1;
 
+    public String readFile(String fileName) {
```

Comment:
we should have only 1 public method - getStatistic
Private ones (methods, classes) should be after public ones.